_posts/blog/2016-07-18-subjects.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: post
+title: "Subjects - Neither Good nor Evil"
+modified:
+categories: blog
+author: tomek_polanski
+excerpt: Get to know the thread switching operations
+tags: [RxJava]
+image:
+date: 2016-07-18T13:00:55-01:00
+---
+
+
+I’ve heard plenty of mixed feelings about RxJava ``Subjects``; some developers I’ve spoken to say they are evil, others say they are the best thing since sliced bacon. 
```

Comment:
the ; behind the ``Subjects``looks a bit strange - looks like java code and it should end the code quote